src/Tests/dotnet-watch.Tests/CommandLineOptionsTests.cs

```diff
@@ -365,7 +365,9 @@ public void OptionDuplicates_NotAllowed(string[] args)
             Assert.Null(options);
             Assert.Equal(1, errorCode);
 
-            Assert.Equal("", output.ToString());
+            // TODO: Re-enable this check when a new version of S.CL is released with this fix in it:
+            // https://github.com/dotnet/command-line-api/pull/2289
+            //Assert.Equal("", output.ToString());
```

Comment:
This pr is closed, can this be re-enabled now?

Reply:
Nope. There hasn't been a release of S.CL since then. The version we're coordinating this merge is from August. That's why I wrote the comment specifically about a version of S.CL needing to be released. :-P

Reply:
Ah yes, I missed the word 'released'.... thank you lol